agenda-services/src/main/java/org/exoplatform/agenda/util/NotificationUtils.java

```diff
@@ -286,15 +283,6 @@ public static final void setNotificationRecipients(IdentityManager identityManag
     String showParticipants = getFullUserName(participants, identityManager);
     notification.with(STORED_PARAMETER_EVENT_ATTENDEES, showParticipants);
 
-    // After computing all usernames, to whom, notifications will be sent,
-    // this deletes the username of modifier/creator user
-    long userIdentityToExclude = StringUtils.equals(typeModification, "ADDED") ? event.getCreatorId() : modifierId;
-    if (userIdentityToExclude > 0) {
-      Identity identityToExclude = identityManager.getIdentity(String.valueOf(userIdentityToExclude));
-      if (identityToExclude != null) {
-        recipients.remove(identityToExclude.getRemoteId());
-      }
-    }
```

Comment:
Why do you remove this part ?
It prevent user modifier to receive a notification about an event he just modify

Reply:
This change was requested in task https://community.exoplatform.com/portal/dw/tasks/taskDetail/73531